README.md

```diff
@@ -28,6 +29,20 @@ If there was no exporter configured or they haven't exported for a while it get
 
 In order to shed some more light in the dark we build a tool called zdb - Zeebe Debugger. It should help you along the way during incidents and broken systems.
 
+
+## Installation
+
+Log into your Zeebe broker machine/container: e.g. `docker exec -it <container_id> /bin/bash`. Then run the following commands (requires `curl` to be installed):
+```bash
+cd /usr/bin
+curl -O -L https://github.com/Zelldon/zdb/releases/download/<version>/zdb
+mkdir target
+curl -o target/zdb-<version>.jar -L https://github.com/Zelldon/zdb/releases/download/<version>/zdb-<version>.jar
```

Comment:
This I would replace with `sed -i 's/target\/// zdb` instead of creating here a subfolder. This sed command replaces the `target/` occurrence and rewrites the file.

Example run:

```
root@87b4a1cdbdd1:/usr/bin# curl -O -L https://github.com/Zelldon/zdb/releases/download/0.5.1/zdb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   611  100   611    0     0   7273      0 --:--:-- --:--:-- --:--:--  7273
100    51  100    51    0     0    316      0 --:--:-- --:--:-- --:--:--  1758
root@87b4a1cdbdd1:/usr/bin# cat zdb 
#!/usr/bin/env bash

java -jar target/zdb-*.jar $@
root@87b4a1cdbdd1:/usr/bin# sed -i 's/target\///' zdb
root@87b4a1cdbdd1:/usr/bin# cat zdb 
#!/usr/bin/env bash

java -jar zdb-*.jar $@
```

Reply:
Makes sense will update it.

Reply:
Done.